tests/contracts/ValidationRulesStorage.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.25;
+
+import "@account-abstraction/contracts/interfaces/IEntryPoint.sol";
+import "./ITestAccount.sol";
+
+contract ValidationRulesStorage is IState {
+    IEntryPoint public entryPoint;
+    uint256 public state;
+
+    function funTSTORE() external override returns(uint256) {
+        assembly {
+            tstore(0, 1)
+        }
+        return 0;
+    }
+
+    function funSSTORE() external returns(uint256) {
+        assembly {
+            sstore(0, 1)
+        }
+        return 0;
+    }
+
+
+    function funTLOAD() external returns(uint256) {
+        uint256 tval;
+        assembly {
+            tval := tload(0)
+        }
+        emit State(tval, tval);
+        return tval;
+    }
+
+    event State(uint oldState, uint newState);
+
+    function setState(uint _state) public {
+        emit State(state, _state);
+        state = _state;
+    }
+
+    function revertOOG() public {
```

Comment:
public functioin generates excessive code, if used in online test.

Reply:
What do you recommend?